the22mastermind/questioner
# Ft user view all meetups 163168045 endpoint

#### What does this PR do?
Add view all meetups functionality with database
#### Description of Task to be completed?
List all registered meetups
#### How should this be manually tested?
1. After cloning repo, cd into it and run 'npm run start'
2. Using POSTMAN enter in URL: http://localhost:5000/api/v1/meetups/ and you should see a list all the meetups registered.
#### Any background context you want to provide?
#### What are the relevant pivotal tracker stories?
#163168045 https://www.pivotaltracker.com/story/show/163168045
#### Screenshots (if appropriate)
#### Questions:
